flagscale/train/train_aquila.py

```diff
@@ -165,12 +165,13 @@ def loss_func(loss_mask: torch.Tensor, output_tensor: torch.Tensor):
 
     losses = output_tensor.float()
     loss_mask = loss_mask.view(-1).float()
+
     total_tokens = loss_mask.sum()
-    loss = torch.cat([torch.sum(losses.view(-1) * loss_mask).view(1), total_tokens.view(1)])
+
+    loss = torch.cat([torch.sum(torch.masked_select(losses.view(-1) , loss_mask==1)).view(1), total_tokens.view(1)])
```

Comment:
This loss will also be used in pre-training. If the SFT requires a different one, we may need a better way to distinguish them.

Reply:
I add a new file called train_aquila_sft.py to distinguish the loss.